### PR TITLE
Test edge case

### PR DESCRIPTION
**Pull request by ferreiratiago**

### CODE REVIEW

**Comment by ferreiratiago** [comment]
### ⛔ **Snyk has found 1 issues.**

| Icon     | Severity | Issues  |
| -------- | -------- | ------- |
|  | Critical | 0 |
|  | High | 0 |
|  | **Medium** | **1** |
|  | Low | 0 |

✅ **security/snyk** check is completed. **No issues were found**. [(View Details)](https://app.dev.snyk.io/org/ferreiratiago/pr-checks/9075e9f4-1f4a-49d0-a4e2-e409a83b88dc?refs=127d45c491f011e9e92da051ef1a722128ff0dc6&source=prChecksComment)

⛔ **code/snyk** check is completed. **Snyk found** `1` **issues**. [(View Details)](https://app.dev.snyk.io/org/ferreiratiago/project/81e7e736-9f50-4f2f-9aa4-c96d9717c0b3/pr-check/b0070d8f-7bef-4781-bb75-73149efd7872?refs=127d45c491f011e9e92da051ef1a722128ff0dc6&source=prChecksComment)

File: app.js [hunk 1]
```diff
@@ -86,3 +97,41 @@ console.log('token: ' + token);
 http.createServer(app).listen(app.get('port'), function () {
   console.log('Express server listening on port ' + app.get('port'));
 });
+
+exports.create = function (req, res, next) {
```
Comment (ferreiratiago) [inline app.js:101]: ### <sub></sub>&nbsp; Allocation of Resources Without Limits or Throttling

This endpoint handler performs a system command execution and does not use a rate-limiting mechanism. It may enable the attackers to perform Denial-of-service attacks. Consider using a rate-limiting middleware such as express-limit.

<h6>
<a href="https://cwe.mitre.org/data/definitions/770.html">CWE-770</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 582</a> | Line 101</h6>

**Review by ferreiratiago — COMMENTED** [review]
## ⚠️&nbsp;Snyk found 1 issue in the modified code


- <sub></sub>&nbsp; **Allocation of Resources Without Limits or Throttling**
  - Path: `app.js#L101`
